gtfs_go_loader.py

```diff
@@ -33,6 +33,9 @@
 from qgis.PyQt import QtWidgets, uic
 from qgis.PyQt.QtCore import QThread, pyqtSignal
 
+from qgis.utils import iface
```

Comment:
I haven't known it is possible to get a ref to QgsInterface by such a simple code, thank you!